### DISCUSSION TITLE
Translation of Django fixtures (django-modeltranslations)

### DISCUSSION THREAD

**Discussion by jojugaad** [WeblateOrg/weblate#7058] [Q&A] [answered]
Hello,

I'm new to Weblate, but think it's a great tool and could be really helpful for me. While I didn't have any difficulties to get it up and running for Django .po and .mo files, as well as VueJS .json locale files, I'm not sure whether it is at all possible to use it also for fixtures which I'm using to prepopulate Django models with translations for django-modeltranslations, like below.

As it's a different structure than standard JSON files this is not compatible with Weblate, is it? I did go through all the docs, but really couldn't find anything at all on this matter...

```json
[
   {
    "model": "energy_sources.energysource",
    "pk": 3,
    "fields": {
      "identifier": "coal",
      "name": "Coal",
      "name_en": "Coal",
      "name_de": "Kohle",
      "alternative_unit": "ton"
    }
  },
  {
    "model": "energy_sources.energysource",
    "pk": 5,
    "fields": {
      "identifier": "district_heating",
      "name": "District heating",
      "name_en": "District heating",
      "name_de": "Fernwärme",
      "alternative_unit": "kWh"
    }
  }
]

**Comment by nijel** [comment, marked as answer]
Weblate currently requires one file per language, so your structure is not supported.

**Reply by jojugaad** [reply]
Alright. Thank you very much for confirming! Then we'll have to put a transformation script in between.
